src/v2/cookbook/adding-instance-properties.md

```diff
@@ -22,32 +22,32 @@ new Vue({
 })
 ```
 
-Then `"My App"` will be logged to the console!
+Entonces `"Mi aplicación"` será mostrado en la consola!
 
-## The Importance of Scoping Instance Properties
+## La importancia de contextualizar las propiedades de instancia
 
-You may be wondering:
+Quizás te estés preguntando:
```

Comment:
It's perfect, just change it to use the formal language type.

For example:

use "se esté preguntando" instead of "te estés preguntando"
use "¿Qué esperaría" instead of "¿Qué esperarías"